### ISSUE TITLE
640gAndroidUploader(0.3) is stopped working

### ISSUE THREAD

**Issue by juliant1dgithub** [pazaan/640gAndroidUploader#64]
#### Your Environment

```
Uploader Version Number: 0.3.0
Your Android Phone Model Name (e.g. Samsung J1 Ace): Moterola G 3nd generation
Android Version Number (e.g. use 4.4 for 4.4.4): 6.0.1
Network Connection at the time the issue occurred (Wi-Fi, Mobile): 3g
(if relevant) Nightscout website version : 0.8.4 Funnel cake, dev
```
#### Brief Explanation of Issue

640gAndroidUploader is crashing  (and this is random) 
(and there are unexplained gaps in the nightscout graph but this is another bug i think)
#### Uploader Error Code

640gAndroidUploader(0.3) is stopped working
#### Steps to Reproduce (for bugs)

unknown 
#### Severity Score

5 = Severe (potential impact on care) -->

**Comment by pazaan** [comment]
Possibly issue #31 in Crashlytics?

**Comment by pazaan** [comment]
Fixed in release 0.3.1
